dataproc/dataproc_quickstart/quickstart.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This sample shows how you can use the Cloud Dataproc Client library to create a
+// Cloud Dataproc cluster
+
+// [START dataproc_quickstart]
+package main
```

Comment:
Does this compile without a `func main()`?

I think the test will run because the testing tool supplies it's own `func main()`, but does `go build` work?

Reply:
Good catch, it does not! Looking into this...

Reply:
Sorry that's not clear. Yes, please switch this to `func main` with flags.